frontend/lib/types/CPtrType.cpp

```diff
@@ -67,9 +87,20 @@ const ID& CPtrType::getId(Context* context) {
   return QUERY_END(result);
 }
 
+const ID& CPtrType::getConstId(Context* context) {
+  QUERY_BEGIN(getConstId, context);
+  UniqueString path = UniqueString::get(context, "CTypes.c_ptrConst");
+  ID result { path, -1, 0 };
+  return QUERY_END(result);
+}
+
 void CPtrType::stringify(std::ostream& ss,
                          chpl::StringifyKind stringKind) const {
-  USTR("c_ptr").stringify(ss, stringKind);
+  if (isConst_) {
+    USTR("c_ptrConst").stringify(ss, stringKind);
```

Comment:
why not `ss << "c_ptrConst"` -- stringifying uniqued strings just prints them no matter what, I believe.

Double checking, I originally wrote this code to use `USTR`. Seems like nonsense to me now.